packages/core/src/indexing-store/cache.ts

```diff
@@ -402,70 +402,72 @@ export const createIndexingCache = ({
         const updateValues = Array.from(updateBuffer.get(table)!.values());
 
         if (insertValues.length > 0) {
-          const text = getCopyText(
-            table,
-            insertValues.map(({ row }) => row),
-          );
-
-          await new Promise(setImmediate);
-
           const endClock = startClock();
 
           // @ts-ignore
           await client.query("SAVEPOINT flush");
-          await copy(table, text)
-            .catch(async (error) => {
-              const result = await recoverBatchError(
-                insertValues,
-                async (values) => {
-                  // @ts-ignore
-                  await client.query("ROLLBACK to flush");
-                  const text = getCopyText(
-                    table,
-                    values.map(({ row }) => row),
-                  );
-                  await copy(table, text);
-                  // @ts-ignore
-                  await client.query("SAVEPOINT flush");
-                },
-              );
 
-              if (result.status === "success") {
-                return;
-              }
+          try {
+            const text = getCopyText(
```

Comment:
@typedarray moving `getCopyText` inside of the try catch improved the log when the error was in that function.

Reply:
Nice, makes sense.